src/chrome/content/rules/Lawyermarketing.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="FindLaw_LawyerMarketing">
```

Comment:
Please avoid underscores in ruleset names.
LGTM otherwise.

Reply:
ping @roanhawkeye

Reply:
Thanks, I will change that :)